src/parser/jobs/rpr/modules/ImmortalSacrifice.tsx

```diff
@@ -0,0 +1,154 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import React from 'react'
+import {DataLink} from '../../../../components/ui/DbLink'
+import {Event, Events} from '../../../../event'
+import {Analyser} from '../../../core/Analyser'
+import {filter} from '../../../core/filter'
+import {dependency} from '../../../core/Injectable'
+import BrokenLog from '../../../core/modules/BrokenLog'
+import Checklist, {Requirement, Rule} from '../../../core/modules/Checklist'
+import {Data} from '../../../core/modules/Data'
```

Comment:
Fixed, this was WebStorm auto path, not sure how to force it to use regular import, have to check that.

Reply:
There's prob some plugin for WebStorm that reads editorconfig files which will enforce most of our styles, but I don't think our linter checks for this either. General recommendation is to use Visual Studio Code (not related to Visual Studio itself) since we have a prebuilt plugin config for it.